src/express/common/config-utils.js

```diff
@@ -129,11 +129,14 @@ export function splitAndGetHostIp(value) {
     }
 }
 
-export async function checkAndReturnVMIndex(n, doc) {
+export async function checkAndReturnVMIndex(n, doc, overrideLog) {
```

Comment:
Hey @Raneet10 I did not quite get the purpose of variable `overrideLog`
Can you elaborate more? Thanks

Reply:
Well I added this because when you don't specify an index, the log prints: `📍Targeting all VMs ...` which isn't accurate for this test as we'll be targeting the first machine. Feel free to suggest if there's a better way to do this!